server/lib/UserManager.php

```diff
@@ -87,7 +87,7 @@ public function search(Request $request, Response $response, array $args = []):
 			return $response->withStatus(404);
 		}
 
-		$search = (string)$params['search'];
+		$search = urldecode((string)$params['search']);
```

Comment:
Isn't it done already? Maybe it is sent encoded, so encoded twice.

Reply:
confirm this is needed as Slim does not parse the array